app/assets/stylesheets/application.scss

```
@@ -2,6 +2,8 @@
 
 @import "./components/miller-columns";
 @import "./components/autocomplete";
+@import "./admin_legacy/govspeak-help";
+@import "./admin/views/govspeak-help";
```

@davidtrussler I think you only need one of these imports here, not both – i.e. not the `admin_legacy` one.

```suggestion
@import "./admin/views/govspeak-help";
```

---

Yes, good point, have removed the unnecessary reference.